pod2daemon/Makefile

```diff
@@ -15,6 +15,16 @@ BUILD_IMAGES    ?=$(FLEXVOL_IMAGE) $(CSI_IMAGE) $(REGISTRAR_IMAGE)
 
 include ../lib.Makefile
 
+# Calculate a timestamp for any build artefacts.
+DATE:=$(shell date -u +'%FT%T%z')
+
+LDFLAGS=-ldflags "\
+	-X $(PACKAGE_NAME)/pkg/lifecycle/startup.VERSION=$(GIT_VERSION) \
```

Comment:
all of these LDFLAGS look to be copied from the node Makefile, which means they aren't going to work here because the package structure isn't the same.

e.g., `pkg/lifecycle/startup.VERSION` doesn't exist in the pod2daemon repository.

I think all we need here is:

```
LDFLAGS=-ldflags "$(RELEASE_LDFLAGS)"
```

Reply:
@caseydavenport One question: Where does the value of the RELEASE_LDFLAGS parameter come from? as It's expected to contain "-w -s".